contracts/Exclusivity.vy

```diff
@@ -1,90 +1,141 @@
 # @version ^0.3.7
-# this relies on functional ActiveUser contract/class
-# no other contract is interfacing with this code as of 12/12 at 9:00AM
```

Comment:
ActiveUser contract is the functionality which checks if the user is in Honors Topics. It already contains a list of Active Users and has methods such as 'getIsActiveUser' or 'getIsAdmin' to check if the user is in a list of existing active honors topics users or is an admin.

Please update your code to Interface with the ActiveUser contract for all related behavior of topicsAddresses